spec/helper/versioning_helper_spec.rb

```diff
@@ -29,6 +58,17 @@
     let(:get_commit_no_items) do
       { body: File.read("#{File.dirname(__FILE__)}/../test_files/get_commit_sha_4ceaceb20e700b92197daf8904f5c4e226625d8a.json") }
     end
+    let(:get_commits_response_hybrid) do
+      { body: File.read("#{File.dirname(__FILE__)}/../test_files/get_commits_since_last_release_hybrid.json") }
```

Comment:
This is a real scenario response obtained when running `purchases-flutter`'s bump lane, I thought it would be better to test using real cases, that's why all the new responses